.github/workflows/binaries.yml

```diff
@@ -9,138 +9,84 @@ on:
         default: 'master'
 
 jobs:
-  linux:
-    runs-on: ubuntu-latest
-    container:
-      image: ocaml/opam2:4.12
-      options: --user root
-    steps:    
-      - name: Checkout stanc3
-        uses: actions/checkout@v2
-        with:
-          ref: ${{ github.event.inputs.branch }}
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macos
+          - ubuntu
+        ocaml-compiler:
+          - 4.12.0
+
+    runs-on: ${{ matrix.os }}-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
 
-      - name: opam caching
-        id: opam-cache
-        uses: actions/cache@v2
+      - name: Use OCaml ${{ matrix.ocaml-compiler }}
+        uses: ocaml/setup-ocaml@v2
         with:
-          path: "~/.opam"
-          key: linux-stanc-binary-4.12.0
+          ocaml-compiler: ${{ matrix.ocaml-compiler }}
+          cache-prefix: v1-windows
```

Comment:
This is the ubuntu/macos build right? What does this cache-prefix name mean here then?